.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contribution Guidelines
+<img src="https://yoast-mercury.s3.amazonaws.com/uploads/2013/02/Yoast_Logo_Large_RGB.png" alt="Yoast logo" width="250px" />
+
+Before filing a bug report or a feature request, be sure to read the contribution guidelines.
+
+## How to use GitHub
+We use GitHub exclusively for well-documented bugs, feature requests and code contributions.
+Please follow the instructions in the issue and PR templates. 
+
+To receive free support for ACF Content Analisys for Yoast SEO we have the following channels:
+* [Support forums](https://wordpress.org/support/plugin/acf-content-analysis-for-yoast-seo/) on WordPress.org
+
+## Security issues
+Please do not report security issues here. Instead, email them to security at yoast dot com so we can deal with them securely and quickly.
+
+## I have found a bug
+Before opening a new issue, please:
+* update to the newest versions of WordPress and ACF Content Analisys for Yoast SEO.
```

Comment:
Typo in "Analisys"

Also we should probably propose updating ACF and Yoast SEO as well. Or do we care about bugs with old versions that much?

Reply:
Good point, I'll add it.